core/client/fs/src/test/java/alluxio/client/block/stream/TestDataWriter.java

```diff
@@ -30,6 +30,7 @@ public TestDataWriter(ByteBuffer buffer) {
   public void writeChunk(ByteBuf chunk) throws IOException {
```

Comment:
@ssz1997 we always try to follow the best practice even if it's in Tests.
Always wrap release() in try{} finally{release} so that no matter what happens in the middle, we always remember to release the chunk